tests/cli/test_utils.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""Tests for the :mod:`aiida_common_workflows.cli.utils` module."""
+import pytest
+
+from aiida_common_workflows.cli.utils import get_code_from_list_or_database
+
+
+@pytest.mark.usefixtures('clear_database_before_test')
+def test_get_code_from_list_or_database(generate_code):
+    """Test `get_code_from_list_or_database` method."""
+    entry_point = 'quantumespresso.pw'
+
+    # No explicit codes nor stored in the database
+    assert get_code_from_list_or_database([], entry_point) is None
+
+    codes = [generate_code(entry_point).store(), generate_code(entry_point).store()]
+
+    # From explicit code list
+    assert get_code_from_list_or_database(codes, entry_point).uuid == codes[0].uuid
+
+    # Empty list, but database contains a code
+    assert get_code_from_list_or_database([], entry_point).uuid == codes[0].uuid
```

Comment:
The problem here is that you generate two codes and you store them. Then in the last assert, you assume you parse with `get_code_from_list_or_database` the first one in the list `codes`, while in my local environment, it happens to parse the second.
Maybe you did not want to store one of the two codes??

Reply:
I create two codes intentionally. The first test case should always return the first element because the list is ordered and so the first code will be matched by the function and returned. The problem is in the second test case where the result will come from the database where order is random. I will update the test to check that either one is returned